### ISSUE TITLE
pylxd 2.2.5 (on bionic) is broken due to lxd 3.0.0-beta{x}

### ISSUE THREAD

**Issue by ajkavanagh** [lxc/pylxd#284]
Unfortunately, on the 21st February (15 days ago), lxd 3.0 great an additional column/attribute called 'description' for operations (see https://github.com/lxc/lxd/commit/7414589c25b631b9a08b150a642748569498f715 for further details).  Sadly, the pylxd operation.py can't handle attributes it doesn't know about, and thus crashes.

It needs a fix similar to the 2.2.4 fix for handling unknown attributes

```bash
45-4a09-9269-d69e5925b1e4] Traceback (most recent call last):
45-4a09-9269-d69e5925b1e4]   File "/usr/lib/python2.7/dist-packages/nova/compute/manager.py", line 2238, in _build_resources
45-4a09-9269-d69e5925b1e4]     yield resources
45-4a09-9269-d69e5925b1e4]   File "/usr/lib/python2.7/dist-packages/nova/compute/manager.py", line 2022, in _build_and_run_instance
45-4a09-9269-d69e5925b1e4]     block_device_info=block_device_info)
45-4a09-9269-d69e5925b1e4]   File "/usr/lib/python2.7/dist-packages/nova/virt/lxd/driver.py", line 509, in spawn
45-4a09-9269-d69e5925b1e4]     self.client, context, instance.image_ref)
45-4a09-9269-d69e5925b1e4]   File "/usr/lib/python2.7/dist-packages/nova/virt/lxd/driver.py", line 351, in _sync_glance_image_to_lxd
45-4a09-9269-d69e5925b1e4]     wait=True)
45-4a09-9269-d69e5925b1e4]   File "/usr/lib/python2.7/dist-packages/pylxd/models/image.py", line 135, in create
45-4a09-9269-d69e5925b1e4]     response.json()['operation'])
45-4a09-9269-d69e5925b1e4]   File "/usr/lib/python2.7/dist-packages/pylxd/models/operation.py", line 28, in wait_for_operation
45-4a09-9269-d69e5925b1e4]     operation = cls.get(client, operation_id)
45-4a09-9269-d69e5925b1e4]   File "/usr/lib/python2.7/dist-packages/pylxd/models/operation.py", line 38, in get
45-4a09-9269-d69e5925b1e4]     return cls(_client=client, **response.json()['metadata'])
45-4a09-9269-d69e5925b1e4]   File "/usr/lib/python2.7/dist-packages/pylxd/models/operation.py", line 43, in __init__
45-4a09-9269-d69e5925b1e4]     setattr(self, key, value)
45-4a09-9269-d69e5925b1e4] AttributeError: 'Operation' object has no attribute 'description'
```

**Comment by ajkavanagh** [comment]
Also, relevant here is that it needs to be backported to xenial, zesty, artful.

**Comment by ajkavanagh** [comment]
This is now fixed in master.

**Comment by ajkavanagh** [comment]
This was released as part of 2.2.6/2.0.7 on March 9th 2018
